index.js

```diff
@@ -107,8 +107,22 @@ let runID = new Date().getTime();
 
 function completer(line) {
   let completions = '/summarize /save /run /quit /explore /assert /undo /manual'.split(' ')
-  
-  completions.concat(tasks)
+  let filesAndFolders = fs.readdirSync('./')
+  filesAndFolders = filesAndFolders.map(file => {
+    let stats = fs.statSync(file)
+    if(stats.isDirectory()){
+      return file + '/' // add a '/' at the end of the folder name
+    } else {
+      return file
+    }
+  })
```

Comment:
this should be nested under the conditional on line 120

Reply:
ya, I would have to move and define a recursive call for this to handle the subdirectories and will later call it inside line 120